etc/telegraf.conf

```diff
@@ -860,6 +860,18 @@
 #   ## Master node.
 #   cluster_stats = false
 #
+#   ## Set indices_stats to true when you want to obtain indices stats from the Master node.
+#   indices_stats = false
+#
+#   ## Set shards_stats to true when you want to obtain shards stats from the Master node.
+#   ## If set, then indices_stats is considered true as they are also provided with shard stats.
+#   shards_stats = false
+
+#   ## Multiplier of the elasticsearch interval to be used to reduce the frequency of
+#   ## indices_stats and shards_stats reports. The default interval is 10 seconds, and this
+#   ## multiplier is defaulted to 6 to cause these metrics to be taken once per minute by default.
+#   indices_interval_multiplier = 6
```

Comment:
Indices stats are a huge volume of data and (at least for our purposes) weren't needed at the same interval as the rest of the ES metrics. The data is detail and important, but generally doesn't fluctuate as much as the other statistics.   

Since our existing on-prem solution used a multiplier,  this solution was consistent for us - and much easier to handle within the plugin than providing a separate interval which may (or may not) be an even multiple of the telegraf ES interval. Say the base interval is set to 10s while this special interval got set to 35s for some odd reason - this seemed a bit awkward. So we went with the multiplier. 

I can update the README accordingly if you like. 

I'd actually submitted these changes late last year, but they needed rebasing (I didn't see your comment to that affect at the time). Took me a while to get around to it. 

Its obviously up to you on which PR provides the most/best data.  This PR hits different endpoints, and provides shardshats as well which I don't think #2456 does.